packaging_lumbar_rootlets/README.md

```diff
@@ -0,0 +1,69 @@
+## Getting started
+
+⚠️ This README provides instructions on how to use the model for **_lumbar_** rootlets. 
+Please note that this model is still under development and is not yet available in the Spinal Cord Toolbox (SCT).
+
+⚠️ For the stable model for dorsal cervical rootlets only, use SCT v6.2 or higher (please refer to this [README](..%2FREADME.md)).
+
+### Dependencies
+
+- [Spinal Cord Toolbox (SCT)](https://spinalcordtoolbox.com/user_section/installation.html)
+- [conda](https://conda.io/projects/conda/en/latest/user-guide/install/index.html) 
+- Python
+
+### Step 1: Cloning the Repository
+
+Open a terminal and clone the repository using the following command:
+
+```
+git clone https://github.com/ivadomed/model-spinal-rootlets
+```
+
+### Step 2: Setting up the Environment
+
+The following commands show how to set up the environment. 
+Note that the documentation assumes that the user has `conda` installed on their system. 
+Instructions on installing `conda` can be found [here](https://conda.io/projects/conda/en/latest/user-guide/install/index.html).
+
+1. Create a conda environment with the following command:
+```
+conda create -n venv_nnunet python=3.9
+```
+
+2. Activate the environment with the following command:
+```
+conda activate venv_nnunet
+```
+
+3. Install the required packages with the following command:
+```
+cd model-spinal-rootlets
+pip install -r packaging_lumbar_rootlets/requirements.txt
+```
+ 
+### Step 3: Getting the Predictions
+
+ℹ️ To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
```

Comment:
```suggestion
> [!NOTE]
> To temporarily suppress warnings raised by the nnUNet, you can run the following three commands in the same terminal session as the above command:
> ```bash
> export nnUNet_raw="${HOME}/nnUNet_raw"
> export nnUNet_preprocessed="${HOME}/nnUNet_preprocessed"
> export nnUNet_results="${HOME}/nnUNet_results"
> ```
```
I'm not sure this formatting is going to work...

Reply:
Yes, it is working! https://github.com/ivadomed/model-spinal-rootlets/blob/main/packaging_lumbar_rootlets/README.md#step-3-getting-the-predictions